package-lock.json

```diff
@@ -221,21 +221,21 @@
       "resolved": "https://registry.npmjs.org/@edx/paragon/-/paragon-3.9.3.tgz",
       "integrity": "sha512-jSBh8D8fCPlR3QALwQQRVXwXJUfyr0ogsI9a+PUAWBVMb4Z0nnKFIMzCWzpG8/H5N4MK3TtsgGX2XWpP0rXFzw==",
       "requires": {
-        "@edx/edx-bootstrap": "^1.0.0",
```

Comment:
Why package-lock.json changes, if you didn't touch package.json?

Reply:
This is not on purpose, I will look into why this happened.